.env

```diff
@@ -1,3 +1,4 @@
+VITE_APP_URL=http://localhost:8080
```

Comment:
I probably missed some, someone really double check me. @Haviles04 @itsalaidbacklife

Reply:
Additional files using localhost:
* `vite.config.mjs`
* `config/env/staging.js`

Places I'm unsure whether we want to change the hard coded references:
* tests - I'd think we want to keep the tests commands e.g. `rankedMatches.spec.js` to use localhost in `cy.request()` to avoid accidentally testing against prod and generating bs data
* `src/plugins/sails.js` - fallback in case of missing env; seems reasonable to keep it

Reply:
We should not change `HOST_SERVER_URL` in `vite.config` as it's used for the docker version, and server only. Same for `config/env/staging.js`, they should stay hardcoded since it's based on hard url location. I also think tests are fine staying hardcoded for that same reason.

Reply:
It's slightly confusing that this is the only place that `VITE_API_URL` and `VITE_APP_URL` differ

Reply:
It's very confusing but has to do with where Cuttle needs these values. The server only ever cares about 1337. The client cares about both.

That's why I said I may have missed something, but this all works from my testing.